cargo-shuttle/src/lib.rs

```diff
@@ -1933,6 +1940,9 @@ impl Shuttle {
 
         let res = client.delete_project(pid).await?.into_inner();
 
+        self.ctx.remove_project_id();
+        self.ctx.save_local_internal()?;
```

Comment:
If a `--id` different from the id in the config file is provided, we don't want to clear the id in the config file.